chart/values.yaml

```
@@ -129,44 +122,68 @@ allowPodLaunching: true
 # Images
 images:
   airflow:
+    # images.airflow.respository is used to override the defaultAirflowRepository for Airflow image
     repository: ~
+    # images.airflow.tag is used to override the defaultAirflowTag for Airflow image
     tag: ~
+    # images.airflow.pullPolicy is the pullPolicy for the Airflow image
     pullPolicy: IfNotPresent
```

```suggestion
```suggestion
  # images.airflow sets the Airflow image details
  airflow:
    repository: ~
    tag: ~
    pullPolicy: IfNotPresent
```

We could strike a compromise for things that are pretty self-explanatory too.